update_twiki.py

```diff
@@ -0,0 +1,109 @@
+"""
+Twiki module: Creates a database for missing samples(too be expanded and completed)
+"""
+import sys
+import sqlite3
+import logging
+import numpy as np
+sys.path.append('/afs/cern.ch/cms/PPD/PdmV/tools/McM/')
+from rest import McM
+#XSDB wrapper
+from request_wrapper import RequestWrapper
+#XSDB pycurl requester instance
+xsdb_request = RequestWrapper()
+# McM instance
+mcm = McM(dev=False, debug=False, cookie='cookie.txt')
+# Logger
+logging.basicConfig(format='[%(asctime)s][%(levelname)s] %(message)s', level=logging.INFO)
+logger = logging.getLogger()
+
+def create_table():
+    conn = sqlite3.connect('twiki.db')
+    c = conn.cursor()
+    # Create table if it does not exist
+    c.execute('''CREATE TABLE IF NOT EXISTS twiki_samples
+                 (prepid text NOT NULL,
+                  dataset text NOT NULL,
+                  extension text,
+                  total_events integer NOT NULL,
+                  campaign text NOT NULL,
+                  resp_group text NOT NULL,
+                  cross_section float NOT NULL,
+                  fraction_negative_weight float NOT NULL,
+                  target_num_events real NOT NULL   )''')
+
+    # Clear the table
+    c.execute('DELETE FROM `twiki_samples`')
+    conn.commit()
+    conn.close()
+def insert_update(twiki_request, campaign, cross_section, frac_neg_wgts, target_num_events):
```

Comment:
While we are not yet at "fixing style" step, for the sake of readability, please add some empty lines in between methods.
Methods that are inside a class should be separated with one empty line.
Methods that are outside classes (like in here) should be separated with two empty lines.